app/scripts/controllers/overview.js

```diff
@@ -385,6 +386,32 @@ angular.module('openshiftConsole')
       $scope.renderOptions.showLoading = !loaded && projectEmpty;
     };
 
+    var setGenericQuotaWarning = function() {
+      var isHidden = AlertMessageService.isAlertPermanentlyHidden("overview-quota-limit-reached", $scope.projectName);
+      if (!isHidden && QuotaService.isAnyQuotaExceeded($scope.quotas, $scope.clusterQuotas)) {
+        $scope.alerts['quotaExceeded'] = {
```

Comment:
Will the alert will reappear on the next poll if I close it with the `x`? If so, does

```js
if ($scope.alerts.quotaExceeded) {
  return;
}
```

prevent that?